packages/search/src/search-service.ts

```diff
@@ -41,25 +42,38 @@ export class SearchService extends SDKService {
   static search(query: Query, config: CrafterConfig): TodoSearchReturnType;
   static search(params: Object, config: CrafterConfig): TodoSearchReturnType;
   static search(queryOrParams: Query | Object, config: CrafterConfig): TodoSearchReturnType {
-    const requestURL = composeUrl(config, crafterConf.getConfig().endpoints.SEARCH),
-          params = (queryOrParams instanceof Query)
+    let requestURL;
+    const params = (queryOrParams instanceof Query)
             ? queryOrParams.params
             : queryOrParams,
           searchParams = new URLSearchParams();
 
-    for (let param in params) {
-      if (Array.isArray(params[param])) {
-        for (let x = 0; x < params[param].length; x++) {
-          searchParams.append(param, params[param][x]);
+    if (queryOrParams instanceof ElasticQuery) {
+      document.cookie = `crafterSite=${ config.site }`;
```

Comment:
Check if we should add path or other cookie directives

Reply:
actually we don't need this cookie anymore, proceeding to remove the line